src/library/Uno.Themes.WinUI.Markup/Theme/TextBox.cs

```diff
@@ -0,0 +1,207 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxPlaceholderForegroundDisabled", true);
+					}
+
+
+					public static class DeleteButtonForeground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForeground")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxDeleteButtonForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxDeleteButtonForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxDeleteButtonForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxDeleteButtonForegroundDisabled", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackground")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxBackgroundDisabled", true);
+
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrush")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxBorderBrushFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxBorderBrushDisabled", true);
+					}
+
+					public static class Typo
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "FilledTextBoxFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("FilledTextBoxFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(FontWeight), "FilledTextBoxFontWeight")]
```

Comment:
I think you will need to have different namespaces for FontWeight @Arieldelossantos (cc @kazo0, @Xiaoy312)

﻿#if WinUI
using Microsoft.UI.Text;
#else
using Windows.UI.Text;
#endif

Reply:
fixed

Reply:
@Arieldelossantos Sorry my bad I confirmed with Steve and Xiaotian
We only support WinUI, so no need to use `#if uwp/winui` in code

Reply:
done